cylc/flow/tests/test_task_trigger.py

```diff
@@ -0,0 +1,86 @@
+import pytest
```

Comment:
Missing license header I think

Reply:
No probs, I'll sort that.

Nice to meet you all too  :)